tests/autoyast/installation.pm

```diff
@@ -1,4 +1,4 @@
-# Copyright (C) 2015-2019 SUSE Linux GmbH
+# Copyright © 2015-2020 SUSE LLC
```

Comment:
Please, put this change and change to `tests/installation/scc_registration.pm` to separate commit, so we don't loose it when revert the workaround.

Reply:
done in separate commit